cabal-testsuite/src/Test/Cabal/Run.hs

```diff
@@ -63,7 +63,8 @@ run _verbosity mb_cwd env_overrides path0 args input = do
           , std_out = UseHandle writeh
           , std_err = UseHandle writeh
           }
-    (stdin_h, _, _, procHandle) <- createProcess prc
+
+    withCreateProcess prc $ \stdin_h _ _ procHandle -> do
```

Comment:
This is required to make sure the spawn process is termined with the Haskell thread is terminated. This is important when we want to run long-running processes in parallel with a part of our code.